team-agreements/definition-of-ready/readme.md

```diff
@@ -16,6 +16,8 @@ It can be understood as a checklist that helps the Product Owner to ensure that
 * [ ] Does the user story have clear and complete acceptance criteria?
 * [ ] Does the user story address the business need?
 * [ ] Can we measure the acceptance criteria?
+* [ ] Can the story be broken down into smaller stories?

```

Comment:
Would suggest changing to more concrete target such as. This will help communicate when a story is small enough.

> Can the story be completed within a single sprint? 

This shouldnt be taken as law (i.e. no story that spans a sprint can be accepted by the team). However, it should be the exception and very intentional decision by the team to take a story that they know will span multiple sprints.